package.json

```diff
@@ -48,15 +50,11 @@
     "esbuild-copy-static-files": "0.1.0",
     "esbuild-plugin-clean": "1.0.1",
     "eslint": "8.35.0",
-    "eslint-config-react-app": "7.0.1",
     "eslint-import-resolver-typescript": "3.5.3",
-    "eslint-plugin-flowtype": "8.0.3",
     "eslint-plugin-import": "2.27.5",
-    "eslint-plugin-jsx-a11y": "6.7.1",
     "git-rev-sync": "3.0.2",
     "jest": "29.4.3",
     "nodemon": "2.0.20",
-    "npm-run-all": "4.1.5",
```

Comment:
This should not be part of this PR.

Reply:
Whoops, committed that to the wrong branch